### DISCUSSION TITLE
Split CSS og svelte dokumentation op?

### DISCUSSION THREAD

**Discussion by tegner** [EkstraBladetUdvikling/eb-designsystem#114] [Ideas]
Skal vi prøve at strukturere dokumentationen så det er åbenlyst hvilke dele der findes som decideret svelte komponenter?

og samtidig indsætte svelte dogmerne

**Comment by tcarlsen** [comment]
mit bud på en løsning: https://github.com/EkstraBladetUdvikling/eb-designsystem/pull/122
